### PR TITLE
LPS-121864 Migrate management toolbar v2 usages in blogs-web

### PR DESCRIPTION
**Pull request by markocikos**
Fixes https://issues.liferay.com/browse/LPS-121864

This PR contains the first 'real' usage upgrade to the new management toolbar tag, based on Clay v3. The only existing usage so far is in `frontend-taglib-clay-sample-web`.

Test plan:
1. Open Content & Data > Blogs. There are two tabs, with two management toolbars, 'Entries' and 'Images'
2. To create entry samples, create from (+) button on entries tab.
3. To create image samples, open one entry, add a bunch of images, save.
4. Confirm that all features follow [Lexicon guidelines](https://liferay.design/lexicon/core-components/toolbars/management-bar/). 
See gif on bottom. 

@liferay-lima Please check all the use cases, it is quite possible that I missed some.

Notes:
- I had to make significant changes in the usages, primarily to enable bulk actions, i.e. deletions. For both toolbars we are implementing `*PropsTransformer`, as a part of standard wiring pattern for React-based components. For entries, this meant rewriting a `DefaultEventHandler`. For images, this meant rewriting JS block in JSP. 
- Images implementation looks older than entries, and there are quite a few additional tweaks that can be made to make it consistent. For example, `postForm` can be used in images form, instead of `submitForm`. I only made those changes that enabled component to fully function. Implementing `SearchContainerManagementToolbarDisplayContext` inheritance for display context is strictly speaking not mandatory, but it makes it easier, as it provides ootb attributes required for interaction with search container. The same is true for adding `managementToolbarDisplayContext` attribute to images; it is useful as it to provides ootb attributes like namespace, but it is up to to you if you want to remove it or collapse other attributes into it.
- `_isTrashEnabled` block is a copy from `BlogsEntryActionDropdownItemsProvider`. I made the change to remove the error throwing, as this clashed with super-class method not throwing an error.
- We are setting `selectable` to be `true` by default, to match v2 default. In addition, we are doing a bit of housekeeping on the tag, to make defaults handling consistent. All non-`null` defaults are now in `ManagementToolbarDefaults`, all `null` defaults have a consistent and simplest check.

/cc @carloslancha @julien @jonmak08 @jbalsas

after

### CODE REVIEW

File: modules/apps/blogs/blogs-web/src/main/resources/META-INF/resources/blogs_admin/view_images.jsp [hunk 1]
```diff
@@ -105,58 +106,4 @@ String displayStyle = blogImagesManagementToolbarDisplayContext.getDisplayStyle(
 			/>
 		</liferay-ui:search-container>
 	</aui:form>
-</clay:container-fluid>
-
-<aui:script>
```
Comment (jbalsas) [inline modules/apps/blogs/blogs-web/src/main/resources/META-INF/resources/blogs_admin/view_images.jsp:110]: Nice! 🤩

**Comment by markocikos** [comment]
ci:test:sf

**Comment by markocikos** [comment]
ci:test:relevant

**Comment by boton** [comment]
I'm checking

**Comment by boton** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by boton** [comment]
LGTM. Thanks @markocikos for `cleaning view_images.jsp` 👏 ❤️

**Comment by boton** [comment]
ci:forward

**Comment by carloslancha** [comment]
@markocikos @boton it looks like we "broke" some poshi tests, we probably need to update them to fit the new markup. Could you mention your Q.A so he/she can help us with it? Thx guys!

**Comment by boton** [comment]
@austinchiang Can you or someone take a look at his failing test?

**Comment by Jeremy-Ch** [comment]
Hey @boton, I have updated the poshi tests and resend a new pull request, please see #1418, close this pr, thanks.

**Pull request closed without merging**